payments/serializers.py

```diff
@@ -0,0 +1,87 @@
+from rest_framework import serializers
+from .models import Credit, Payment
+from .utils import get_payment_values, add_to_month
+from products.models import Product
+from datetime import datetime
+from decimal import Decimal
+
+class CreditCreationSerializer(serializers.ModelSerializer):
```

Comment:
Your serializer should take fewer responsibilities. Try to delegate more responsibilities to the model.